clients/src/test/java/org/apache/kafka/clients/admin/KafkaAdminClientTest.java

```diff
@@ -225,27 +225,34 @@ public void testInvalidTopicNames() throws Exception {
             env.kafkaClient().setNode(env.cluster().controller());
 
             List<String> sillyTopicNames = Arrays.asList(new String[] {"", null});
+            int originRequestCount = env.kafkaClient().requestCount();
             Map<String, KafkaFuture<Void>> deleteFutures =
                 env.adminClient().deleteTopics(sillyTopicNames).values();
             for (String sillyTopicName : sillyTopicNames) {
                 assertFutureError(deleteFutures.get(sillyTopicName), InvalidTopicException.class);
             }
+            assertEquals(originRequestCount, env.kafkaClient().requestCount());
```

Comment:
Maybe we could make this assertion precise? I think we expect the request count to be 0.

Reply:
Ya, the expected value of count is zero currently. However, we can't assume the initial count won't be changed in the future. For example, we plan to share the AdminClientUnitTestEnv (or MockClient) between all UTs to speedup the tests.

Reply:
Not sure I follow. What would be the benefit of sharing a MockClient?

Reply:
bq. What would be the benefit of sharing a MockClient?
Just saying (smile) 

I'm fine to change the assertion.